pkg/netmap/container.go

```diff
@@ -0,0 +1,19 @@
+package netmap
+
+// Container represents container.
+type Container interface {
```

Comment:
Should we rename this to `ContainerNodes` to avoid confusion with container type?

Reply:
Fixed.